ports/openmvg/portfile.cmake

```
@@ -76,14 +76,7 @@ vcpkg_cmake_configure(
 
 vcpkg_cmake_install()
 
-file(MAKE_DIRECTORY "${CURRENT_PACKAGES_DIR}/share/openMVG/")
-file(RENAME "${CURRENT_PACKAGES_DIR}/lib/openMVG/cmake" "${CURRENT_PACKAGES_DIR}/share/openMVG/cmake")
-if(NOT VCPKG_BUILD_TYPE)
-  file(MAKE_DIRECTORY "${CURRENT_PACKAGES_DIR}/debug/share/openMVG/")
-  file(RENAME "${CURRENT_PACKAGES_DIR}/debug/lib/openMVG/cmake" "${CURRENT_PACKAGES_DIR}/debug/share/openMVG/cmake")
-endif()
-
-vcpkg_cmake_config_fixup()
+vcpkg_cmake_config_fixup(PACKAGE_NAME openMVG CONFIG_PATH lib/openMVG/cmake)
```

```suggestion
vcpkg_cmake_config_fixup(CONFIG_PATH lib/openMVG/cmake)
```
PACKAGE_NAME only affects the chosen directory name. *CMake* is case-insensitive with regard to the directory name. We want the config to live in the lower-case directory `openmvg` which matches the port name. This is also the default PACKAGE_NAME.